frontend/css/wi-volunteer-management-public.css

```diff
@@ -80,4 +80,33 @@
 
 #wivm-sign-up-form .field-error {
 	border: 2px solid #990000;
+}
+
+/* Widget Styles */
+.widget-volunteer-opp-info {
+   max-height: 150px;
+   overflow-y: hidden;
+}
+
+.widget-volunteer-opp-info:hover {
+   overflow-y: auto;
+}
+
+/*
+* Show scrollbar on hover without having to start scrolling so user can tell there are more opportunities
+*
+* http://stackoverflow.com/questions/7855590/how-can-i-prevent-scroll-bars-from-being-hidden-for-os-x-trackpad-users-in-webki 
+*/
+.widget-volunteer-opp-info::-webkit-scrollbar {
```

Comment:
Really nice customization here as well. Like I mentioned, I think we can remove the scrolling here as well.